packages/angular/src/executors/delegate-build/delegate-build.impl.ts

```diff
@@ -0,0 +1,53 @@
+import type { ExecutorContext } from '@nrwl/devkit';
+import {
+  joinPathFragments,
+  parseTargetString,
+  runExecutor,
+} from '@nrwl/devkit';
+import { createProjectGraph } from '@nrwl/workspace/src/core/project-graph';
+import {
+  calculateProjectDependencies,
+  checkDependentProjectsHaveBeenBuilt,
+  createTmpTsConfig,
+} from '@nrwl/workspace/src/utilities/buildable-libs-utils';
+import type { DelegateBuildExecutorSchema } from './schema';
+
+export async function* delegateBuildExecutor(
```

Comment:
I'm curious why this needs to be an iterable function. Could you explain the reasoning behind it?

Reply:
`runExecutor` isn't a generator right?

```ts
export async function runExecutor<T extends { success: boolean }>(
  targetDescription: {
    project: string;
    target: string;
    configuration?: string;
  },
  options: { [k: string]: any },
  context: ExecutorContext
): Promise<AsyncIterableIterator<T>> {
  return await runExecutorInternal<T>(
    targetDescription,
    options,
    context.root,
    context.cwd,
    context.workspace,
    context.isVerbose,
    false
  );
}
```